### ISSUE TITLE
REST Service not working - Pageable could not be constructed

### ISSUE THREAD

**Issue by jdiazgon** [devonfw/devon4j#15]
### Expected behavior

I'm trying to use the new devon4j 3.0.0 archetype with REST services generated by CobiGen. My front-end is an Angular application that tries to get all the employees from the database. The request is the following: `/services/rest/employeemanagement/v1/employee/search`.

I would like to get the correct data from the server without any kind of error.

### Actual behavior

The response from the server is 500: 
```Java
com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `org.springframework.data.domain.Pageable` (no Creators, like default construct, exist): abstract types either need to be mapped to concrete types, have custom deserializer, or contain additional type information
 at [Source: (org.apache.cxf.transport.http.AbstractHTTPDestination$1); line: 1, column: 13] (through reference chain: com.devonfw.poc.angular.employeemanagement.common.api.to.EmployeeSearchCriteriaTo["pageable"])
```

For creating the CobiGen templates, I have followed the [starters example](https://github.com/devonfw/devon4j/blob/develop/starters/starter-spring-data-jpa/src/test/java/com/devonfw/example/component/common/api/to/SearchCriteriaTo.java) and generated a new SearchCriteriaTo on my `common/api/to` folder. That file contains a Pageable attribute that is not being properly constructed as the above error mentions.

Most probably the solution is to provide to the mapper a hint like explained [here](https://stackoverflow.com/questions/29321901/custom-object-deserializes-fine-in-jax-rs-but-if-it-is-used-in-another-object-it). Maybe there is an easy solution I don't see, but I think this is a crosscutting issue to be solved by the devon4j.

**Comment by hohwille** [comment]
@jdiazgon thanks for your testing and feeback. I am curious that just after having devon4j in place for a few days you already come up with such detailed stuff. This is great, I love it.
In general everything that is supposed to be mapped over the wire needs some serializer and deserializer for Jackson if it is not following the Java bean standard.
It is fully fair to expect devon4j to be shipped with that OOTB since we are pushing towards springs pagination support. So, yes, we urgently have to add this stuff and should have it for our first `3.0.0` release.

**Comment by hohwille** [comment]
This fix would need to be added to devon4j:
https://github.com/oasp/oasp4j/pull/694

**Comment by hohwille** [comment]
Merged PR #29, hence done.
